src/app/shared/dialog-box-confirmation/dialog-box-confirmation.component.scss

```diff
@@ -0,0 +1,14 @@
+.cancel {
```

Comment:
let's move that into global.scss and make it a partial please
It seems in fact we might have different styles for different modals.
_dialog-box-confirmation.scss at the root where _global.scss is

Then we load it in the _global.scss in the relevant section